arches/install/arches-templates/project_name/webpack/webpack.common.js

```diff
@@ -33,35 +31,44 @@ module.exports = () => {
             const WEBPACK_DEVELOPMENT_SERVER_PORT = parsedData['WEBPACK_DEVELOPMENT_SERVER_PORT']
 
             // BEGIN create entry point configurations
-        
-            const archesCoreEntryPointConfiguration = buildJavascriptFilepathLookup(Path.resolve(__dirname, ROOT_DIR, 'app', 'media', 'js'), {});
-            const projectEntryPointConfiguration = buildJavascriptFilepathLookup(Path.resolve(__dirname, APP_ROOT, 'media', 'js'), {});
 
-            const archesApplicationsEntrypointConfiguration = ARCHES_APPLICATIONS.reduce((acc, archesApplication) => {   
+            const archesCoreEntryPointConfiguration = buildFilePathLookup(Path.resolve(__dirname, ROOT_DIR, 'app', 'media', 'js'));
+            const projectEntryPointConfiguration = buildFilePathLookup(Path.resolve(__dirname, APP_ROOT, 'media', 'js'));
+
+            const archesApplicationsEntrypointConfiguration = ARCHES_APPLICATIONS.reduce((acc, archesApplication) => {
                 return {
                     ...acc,
-                    ...buildJavascriptFilepathLookup(Path.resolve(__dirname, ARCHES_APPLICATIONS_PATHS[archesApplication], 'media', 'js'), {})
+                    ...buildFilePathLookup(Path.resolve(__dirname, ARCHES_APPLICATIONS_PATHS[archesApplication], 'media', 'js'))
                 };
             }, {});
 
             // END create entry point configurations
             // BEGIN create JavaScript filepath lookups
 
             const archesCoreJavascriptRelativeFilepathToAbsoluteFilepathLookup = Object.entries(archesCoreEntryPointConfiguration).reduce((acc, [path, config]) => {
-                acc[path + '$'] = Path.resolve(__dirname, path, config['import']);
+                if ((typeof config) == "object")
+                    acc[path + '$'] = Path.resolve(__dirname, path, config['import']);
+                else
```

Comment:
how were you able to hit these else conditionals?

Reply:
This happens if there's a non-js file in the `js/` tree. The object type that comes back is dependent on the extension (js files have an object config with name/import keys, the rest are straight strings). I was actually wondering if we should make the value returned from the file lookup builder consistent for all asset types, which would eliminate having to do this here, and make things more obvious from the caller.

Reply:
Yeah I'd definitely say it's would make things neater and more readable, if it doesn't look too complex to pull off 👍

Reply:
When implementing this in 7.6 I did not make these changes and everything still built correctly. Happy to resolve the differences between branches either way and just wanted to flag this 🏴